### PR TITLE
Move Offset by 1 when kafka consumer poll error

### PR DESCRIPTION
**Pull request by alyiwang**

### CODE REVIEW

File: wherehows-kafka/src/main/java/wherehows/actors/KafkaWorker.java [hunk 1]
```diff
@@ -30,28 +32,35 @@
...
   private final String _topic;
 
+  private TopicPartition _partition;  // assuming there is only one partition assigned to each worker
+
   private final KafkaConsumer<String, IndexedRecord> _consumer;
 
   private final KafkaMessageProcessor _processor;
 
-  private final int _consumer_poll_interval = 1000;
+  private boolean _alive;  // status of this worker
```
Comment (mars-lan) [inline wherehows-kafka/src/main/java/wherehows/actors/KafkaWorker.java]: Why do we need this?

File: wherehows-kafka/src/main/java/wherehows/actors/KafkaWorker.java [hunk 2]
```diff
@@ -69,26 +78,34 @@ public void onReceive(Object message) throws Exception {
...
         }
       }
     } else {
       unhandled(message);
     }
   }
 
+  private void moveOffset(int amount) {
+    // Assuming only one partition! Can't get partition info from ConsumerRecords if poll exception.
+    _partition = new TopicPartition(_topic, _consumer.partitionsFor(_topic).get(0).partition());
```
Comment (mars-lan) [inline wherehows-kafka/src/main/java/wherehows/actors/KafkaWorker.java]: How about throwing a RuntimeException if there's more than 1 partition?

File: wherehows-kafka/src/main/java/wherehows/actors/KafkaWorker.java (hunk 1 above)
Comment (alyiwang) [reply wherehows-kafka/src/main/java/wherehows/actors/KafkaWorker.java]: It's not used for now. I though maybe the master would want to check the status of each worker and restart a certain consumer if necessary. I will take this our. We may add this status check or heartbeat later on.

File: wherehows-kafka/src/main/java/wherehows/actors/KafkaWorker.java (hunk 2 above)
Comment (alyiwang) [reply wherehows-kafka/src/main/java/wherehows/actors/KafkaWorker.java]: done

**Pull request merged**